index.html

```diff
@@ -45,24 +38,38 @@ <h2>Experience</h2>
     <li>2003</li>
     <li>2000</li>
 </ul>
-    <ul> 
+<ul>
     <li>TJMax</li>
     <li>Scenthound</li>
-    <li>LLC</li>
-    <li>Military</li>
-    </ul>
+    <li>LLC Isaiko</li>
+    <li>LLC Kokhanovskiy</li>
+</ul>
+<ul>
+    <li>Backroom assosiate</li>
+    <li>Groomer</li>
+    <li>Assistent photographer</li>
+    <li>Assistent web developer</li>
+</ul>
 </div>
 </section>
+<section id="skills">
+    <h2>Skills</h2>
+    <ul>
+        <li>HTML</li>
+        <li>CSS</li>
+        <li>JavaScript</li>
```

Comment:
Since your script will add your skills dynamically, you can leave the `ul` empty to avoid duplicating the items.